soh/soh/GameMenuBar.cpp

```diff
@@ -617,6 +617,15 @@ namespace GameMenuBar {
                         ImGui::EndMenu();
                     }
 
+                    UIWidgets::Spacer(0);
+
+                    bool disabled = !CVarGetInteger("gSuperSonic", 0);
+                    const char* disabledTooltip = "This option is disabled because \"Super Sonic Mode\" is turned off";
```

Comment:
these _could_ be inlined I think, or should be renamed to something less generic, high probability of someone trying to redefine these later on accident.